### ISSUE TITLE
[Visual Studio Code - Project Manager] Rename Extension and Update Icon

### ISSUE THREAD

**Issue by staffanmowitz** [raycast/extensions#15663]
### Extension

https://www.raycast.com/MarkusLanger/vscode-project-manager

### Description

Now that the extension supports all VS Code-based editors, I propose renaming it to something more generic, like "Code Project Manager." 

This would better reflect its broader compatibility beyond just Visual Studio Code. As a user of Cursor who doesn't even have VS Code installed, it's confusing to see "Visual Studio Code" in the "Search Project Manager" command. A more neutral naming would enhance clarity and usability across different editors.

Additionally, I recommend updating the icon to represent the Project Manager extension itself rather than using the VS Code icon. This would provide a more cohesive and representative visual identity.

The included image shows my local forked version of the extension, where I've implemented a new name and icon to illustrate the proposed changes.

![Screenshot 2024-12-03 at 10 48 41@2x](https://github.com/user-attachments/assets/cf87bb6c-5b27-4168-a887-c903c26bd08e)

### Who will benefit from this feature?

All users that does not use VS Code

**Comment by theherk** [comment]
I no longer use this extension, since I'm using [Zoxide Git Projects](https://www.raycast.com/theherk/zoxide-git-projects), so I don't feel strongly one way or the other. I do get your point, and there is definitely some merit. The trouble is that this makes it seems generic and generally usable beyond VS Code derivatives which it isn't. It feels something should specify that, but maybe that isn't necessary.

**Comment by MarkusLanger** [comment]
Good idea. Is it possible to rename an extension name? Because "vscode-project-manger" is then no longer correct either.

**Comment by zhenpewu** [comment]
1. Cursor is a fork of VSCode
2. This extension is based on this VSCode extension instead https://github.com/alefragnani/vscode-project-manager, Don't think it make sense to rename it, as it is better to align with the VSCode extension.
